waspc/src/Wasp/Psl/Generator/Schema.hs

```diff
@@ -0,0 +1,21 @@
+module Wasp.Psl.Generator.Schema
+  ( generateSchemaBlock,
+  )
+where
+
+import qualified Wasp.Psl.Ast.ConfigBlock as Psl.ConfigBlock
+import qualified Wasp.Psl.Ast.Enum as Psl.Enum
+import qualified Wasp.Psl.Ast.Model as Psl.Model
+import qualified Wasp.Psl.Ast.Schema as Psl.Schema
+import Wasp.Psl.Generator.ConfigBlock (generateConfigBlockKeyValues)
+import Wasp.Psl.Generator.Enum (generateEnumBody)
+import Wasp.Psl.Generator.Model (generateModelBody)
+
+type PslSource = String
```

Comment:
This is nice! You should also import it in other Gneerator files and use it. Cyclic imports? Take it out into Generator.Common .